pkgs/top-level/aliases.nix

```
@@ -78,6 +78,7 @@ mapAliases ({
   bittorrentSync = throw "bittorrentSync has been deprecated by resilio-sync."; # added 2019-06-03
   bittorrentSync14 = throw "bittorrentSync14 has been deprecated by resilio-sync."; # added 2019-06-03
   bittorrentSync20 = throw "bittorrentSync20 has been deprecated by resilio-sync."; # added 2019-06-03
+  bs1770gain = throw "bs1770gain has been removed from nixpkgs, as it distributed hate speech."; # added 2021-01-02
```

```suggestion
  bs1770gain = throw "bs1770gain has been removed from nixpkgs."; # added 2021-01-02
```
Lets just leave it like this.